### PR TITLE
Fix cookie message

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Cookie JS isnt working because it replies on the legacy GOV.UK Modules

### Changes proposed in this pull request
Remove dependancy on GOV.UK Modules

### Guidance to review
- Check for errors in browser console.
- Check for a cookie called `seen_cookie_message` and if present then remove it
- Reload the page and check for cookie again, if present then re-load and the message should go

### CODE REVIEW

File: src/package-lock.json [hunk 1]
```diff
@@ -618,12 +618,6 @@
         }
       }
     },
-    "assert-plus": {
```
Comment (fofr) [inline src/package-lock.json]: Why has this commit got a package-lock change?

**Comment by fofr** [comment]
What does the refactoring in aa7cfab074e17245d64a2dfa45079be250c64146 improve? Why is it in this PR?

File: src/package-lock.json (hunk 1 above)
Comment (dankmitchell) [reply src/package-lock.json]: I ran `npm audit fix`

**Comment by dankmitchell** [comment]
@fofr refactored into constructors to make testing easier in the future and consistent with govuk-frontend

File: src/Assets/app.js [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
+import Toggle from './Javascript/toggle';
 import './Javascript/typeahead.jquery.js';
 import './Styles/site.scss';
 
 initAll();
 
-var $backLink = document.querySelector('[data-module="back-link"]')
-new BackLink($backLink).init()
+var $cookieMessage = document.querySelector('[data-module="cookie-message"]');
+new CookieMessage($cookieMessage).init();
```
Comment (fofr) [inline src/Assets/app.js]: All this boilerplate initialisation would be improved with a pattern like https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#modules

**Comment by fofr** [comment]
We need some Javascript tests. That shouldn't block this PR.

File: src/package-lock.json (hunk 1 above)
Comment (fofr) [reply src/package-lock.json]: I'd probably put that in a separate commit with its own commit message explaining.

**Pull request merged**